lib/money/currency.rb

```diff
@@ -14,7 +14,14 @@ class << self
       def new(currency_iso)
         raise UnknownCurrency, "Currency can't be blank" if currency_iso.nil? || currency_iso.empty?
         iso = currency_iso.to_s.downcase
-        @@loaded_currencies[iso] || @@mutex.synchronize { @@loaded_currencies[iso] = super(iso) }
+        @@loaded_currencies[iso] || @@mutex.synchronize do
+          @@loaded_currencies[iso] =
+            if iso == 'xxx'
+              NullCurrency.new
+            else
+              super(iso)
+            end
+        end
```

Comment:
Easier to read but we'd lose memoization. Discussed with Krystian and agreed that it would be better to move this logic to the `value_to_currency` helper since a `.new` does not usually return nil (or in this case our null object).